source/documentation/appendix-docs/conda.md

```diff
@@ -1,89 +1,73 @@
-# Conda Package Management
+# Package management
 
-## Unified package management for both R and Python
+## Introduction
 
-### Faster builds
+A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis. There are a number of package management tools available for both R and Python. Within the MoJ we recommend primarily using [Conda](#Conda) for both languages.
```

Comment:
Do we recommend using Conda for Python too? I confess that I can't remember. I tend to use pip for my JupyterLab on the AP.

Reply:
this was just a guess based on my limited experience - I haven't done nearly as much python, and am not sure what the actual standard is in DASD. I believe what's actually the case is that Conda is specifically an R solution because of the inadequacies of Packrat, while Python has a variety of package management solutions (of which conda is just one).

Reply:
I've now updated to clarify

Reply:
I think we still just recommend `pip` for Python-only projects as conda is not fully supported in JupyterLab – from time to time, you may run into some issues.